pkg/lang/csharp/queries/structural/method_declarations.scm

```diff
@@ -0,0 +1,5 @@
+(method_declaration ;;; [public ...] void myMethod(T1 p1[, ...]){...}
+  type: (_) @return_type
```

Comment:
👍 I like the comment example. Would it be possible to add some to the other queries?